src/components/tooltip/tooltip.spec.js

```diff
@@ -0,0 +1,269 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { render, fireEvent } from '../../test-utils';
+import Tooltip from './tooltip';
+
+jest.mock('popper.js');
+jest.useFakeTimers();
+
+class TestComponent extends React.Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    buttonLabel: PropTypes.string.isRequired,
+    isOpen: PropTypes.bool,
+    id: PropTypes.string,
+    onOpen: PropTypes.func,
+    onClose: PropTypes.func,
+    onFocus: PropTypes.func,
+    onBlur: PropTypes.func,
+    leaveDelay: PropTypes.number,
+    onMouseLeave: PropTypes.func,
+    onMouseOver: PropTypes.func,
+    components: PropTypes.shape({
+      WrapperComponent: PropTypes.any,
+    }),
+  };
+
+  static defaultProps = {
+    title: 'What kind of bear is best?',
```

Comment:
Is this some kind of Canadian thing to say? 😅

Reply:
@emmenko [here](https://www.youtube.com/watch?v=WaaANll8h18)